### DISCUSSION TITLE
How much full node connections need to be?

### DISCUSSION THREAD

**Discussion by SirVladislav** [Chia-Network/chia-blockchain#5523] [Support]
How many full node connections need to be for correctly farming. Now I got near 4 on the main device and 8 in the second harvester that connected to the main. Read that people have 50 and more. Does it increase the chance to win a block? If yes, how to add more connections?

**Comment by BrandtH22** [comment]
@SirVladislav  , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.

Note: I recommend reviewing the farming on multiple machines setup as only one of the machines in a LAN should be acting a full node, having more than one full node in a LAN without strictly setting up the network to handle more than one will lead to both machines having fewer peers and likely experiencing connection issues:
https://docs.chia.net/farming-on-many-machines/
